mixins/button-toggle.scss

```diff
@@ -0,0 +1,124 @@
+@import '../utility';
+
+@mixin blui-mat-button-toggle-default($palette) {
```

Comment:
I like the usage of mixins here but all three seem very similar to each other.  I'm looking at this now to see if we can simplify the logic a little bit.

Reply:
Please review.